docs/source/models/gbd2017/risks/high_ldl_cholesterol/index.rst

```diff
@@ -0,0 +1,120 @@
+.. _2017_risk_high_ldl_c:
+
+====================
+High LDL cholesterol
+====================
+
+Risk Exposure Description
+-------------------------
+
+.. todo::
+
+   Add more information and references. In particular, find data about global prevalence and relation to disease fatal and non-fatal description.
+
+Low-density lipoprotein (LDL) is one of the five major groups of lipoprotein which transport all fat molecules around the body in the extracellular water. LDL can contribute to atherosclerosis if it is oxidized within the walls of arteries.
+
```

Comment:
Wise to differentiate LDL-C from total cholesterol here?  Both appear in literature together quite often, and care should be taken when considering data, etc.

Reply:
If there is something specific you recommend to include with references I should cite, let me know and I can add! I did not include this differentiation yet because I wouldn't know how to cite this.